test/JDBC/jdbc_schedule

```diff
@@ -52,6 +52,8 @@ ignore#!#BABEL-CHECK-CONSTRAINT-before-14_6-vu-verify
 ignore#!#BABEL-CHECK-CONSTRAINT-before-14_6-vu-cleanup
 ignore#!#BABEL-3646-before-14_6-vu-prepare
 ignore#!#BABEL-3646-before-14_6-vu-verify
+ignore#!#openjson-vu-prepare
+ignore#!#openjson-vu-verify
```

Comment:
I thought `openjson` was a basic test file that could be run in the latest as well as some previous versions. Why do we need to ignore it?

Reply:
You're right will remove this